setup.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+from setuptools import setup
+
+from slconf import __version__
+
+# Utility function to read the README file.
+# Used for the long_description.  It's nice, because now 1) we have a top level
+# README file and 2) it's easier to type in the README file than to put a raw
+# string in below ...
+
+
+def read(fname):
+    return open(Path(__file__).parent / fname).read()
+
+
+def read_requirements(filename):
+    with open(filename) as f:
+        return f.read().splitlines()
+
+
+settings = dict(
+    name='slconf',
+    packages=['slconf'],
+    version=__version__,
+    author='cameronwaterman',
+    author_email='cameron.waterman@ni.com',
```

Comment:
Based on the Python clients:

```suggestion
    author="National Instruments",
    maintainer="Cameron Waterman",
    maintainer_email="cameron.waterman@ni.com",
```